Backend/app/rider-platform/rider-app/Main/src/Beckn/ACL/Search.hs

```diff
@@ -174,6 +178,12 @@ mkIntent origin destination customerLanguage distance duration mbPoints = do
                   code = (\_ -> Just "customer_language") =<< customerLanguage,
                   name = (\_ -> Just "Customer Language") =<< customerLanguage,
                   value = (Just . show) =<< customerLanguage
+                },
+              Search.Tag
+                { display = (\_ -> Just False) =<< customerLanguage,
```

Comment:
It should be `disabilityTag` instead of `customerLanguage` here too, right?

Reply:
Yeah